src/main/java/seedu/java/util/Parser.java

```diff
@@ -0,0 +1,74 @@
+package seedu.java.util;
+
+/**
+ * All static method. Handles input from user.
+ * readCommand(String)
+ * readTask(String)
+ * readTiming(String)
+ * readNum(String)
+ */
+public class Parser {
+    /**
+     * Static method. Returns a Command (inferred instruction)
+     * @param input from UI
+     * @return Command
+     * @throws Exception
+     */
+    public static Command readCommand(String input) throws Exception {
+        return Command.convert(input.split(" ")[0]);
+    }
+
+    /**
+     * Reads user's input and returns a String.
+     * @param input - from UI
+     * @return String - task that the user wants to do
+     * @throws Exception - when the parser fails to interpret the task
+     */
+    public static String readTask(String input) throws Exception  {
+        String[] arr = input.split(" ");
+        String task = "";
+        int i = 1;
+        while (i < arr.length && !arr[i].startsWith("/"))  {
+            task += arr[i++] + " ";
+        }
+        if (task.equals("")) {
+            throw new Exception();
+        } else {
+            return task;
+        }
+    }
+
+    /**
+     * Reads user's input and returns a timing in the form of a string.
```

Comment:
To me, it is also not very clear what this method does from the Javadocs. It looks quite similar to `readTask`, so maybe you could include a line on how this method is different from `readTask`?